source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -341,12 +348,12 @@ namespace PlayFab
 
         if (playFabEmitRequest->callback != nullptr)
         {
-            playFabEmitRequest->callback(playFabEmitRequest->event, std::move(response));
+            playFabEmitRequest->callback(playFabEmitRequest->event, response);
         }
 
         if (playFabEmitRequest->stdCallback != nullptr)
```

Comment:
>stdCallback [](start = 32, length = 11)

what is the expectation around the use of `playFabEmitRequest->callback` vs `playFabEmitRequest->stdCallback` is there ever a world where a title would want to set both? is that intentionally permitted by your API?